src/treelearner/cost_effective_gradient_boosting.hpp

```diff
@@ -0,0 +1,111 @@
+/*!
+ * Copyright (c) 2016 Microsoft Corporation. All rights reserved.
```

Comment:
I guess it should be `2019`. Refer to https://github.com/microsoft/LightGBM/pull/2101#issue-270098443.